frontend/src/component/admin/instance-admin/InstanceAdmin.tsx

```diff
@@ -0,0 +1,11 @@
+import AdminMenu from '../menu/AdminMenu';
+import { InstanceStats } from './InstanceStats/InstanceStats';
+
+export const InstanceAdmin = () => {
+    return (
+        <div>
```

Comment:
Should InstanceStats be available for everyone in the system? Currently we don't gate the stats for Admins only.

Reply:
That was the intention, all users with access should be able to access it for now, as there is not secrets here. All stats will be implicitly available for read only users anyway (number of projects, feature toggles etc).